docs/root/start/quick-start/run-envoy.rst

```diff
@@ -158,3 +158,108 @@ The Envoy admin endpoint should also be available at http://localhost:9901
+or restarting the service, or making any network connections.
+
+If the configuration is valid the process will print ``OK`` and exit with a return code of ``0``.
+
+For invalid configuration the process will print the errors and exit with ``1``.
+
+.. tabs::
+
+   .. tab:: System
+
+      .. code-block:: console
+
+	 $ envoy --mode validate -c my-envoy-config.yaml
+	 [2020-11-08 12:36:06.543][11][info][main] [source/server/server.cc:583] runtime: layers:
+	 - name: base
+	   static_layer:
+	     {}
+	 - name: admin
+	   admin_layer:
+	     {}
+	 [2020-11-08 12:36:06.543][11][info][config] [source/server/configuration_impl.cc:95] loading tracing configuration
+	 [2020-11-08 12:36:06.543][11][info][config] [source/server/configuration_impl.cc:70] loading 0 static secret(s)
+	 [2020-11-08 12:36:06.543][11][info][config] [source/server/configuration_impl.cc:76] loading 1 cluster(s)
+	 [2020-11-08 12:36:06.546][11][info][config] [source/server/configuration_impl.cc:80] loading 1 listener(s)
+	 [2020-11-08 12:36:06.549][11][info][config] [source/server/configuration_impl.cc:121] loading stats sink configuration
+	 configuration 'my-envoy-config.yaml' OK
+
+   .. tab:: Docker
+
+      .. substitution-code-block:: console
+
+	 $ docker run --rm \
+	       -v $(pwd)/my-envoy-config.yaml:/my-envoy-config.yaml \
+	       envoyproxy/|envoy_docker_image| \
+	       --mode validate \
+	       -c my-envoy-config.yaml
+	 [2020-11-08 12:36:06.543][11][info][main] [source/server/server.cc:583] runtime: layers:
+	 - name: base
+	   static_layer:
+	     {}
+	 - name: admin
+	   admin_layer:
+	     {}
+	 [2020-11-08 12:36:06.543][11][info][config] [source/server/configuration_impl.cc:95] loading tracing configuration
+	 [2020-11-08 12:36:06.543][11][info][config] [source/server/configuration_impl.cc:70] loading 0 static secret(s)
+	 [2020-11-08 12:36:06.543][11][info][config] [source/server/configuration_impl.cc:76] loading 1 cluster(s)
+	 [2020-11-08 12:36:06.546][11][info][config] [source/server/configuration_impl.cc:80] loading 1 listener(s)
+	 [2020-11-08 12:36:06.549][11][info][config] [source/server/configuration_impl.cc:121] loading stats sink configuration
+	 configuration 'my-envoy-config.yaml' OK
+
+Envoy logging
+-------------
+
+By default Envoy system logs are sent to ``/dev/stderr``.
+
+This can be overridden using ``--log-path``
+
+Access log paths can be set for the admin interface, and for configured listeners.
+
+Some filters may have additional logging capabilities.
+
+Envoy debugging
+---------------
+
+The default log level for Envoy system logs is ``info``.
+
+This can be set using the :option:`-l or --log-level <--log-level>` option.
+
+The available log levels are:
+
+- trace
+- debug
+- info
+- warning/warn
+- error
+- critical
+- off
+
+You can also set the log level for specific components using the :option:`--component-log-level` option.
```

Comment:
it would be great if there were a way to dump the component list without having to read the header file.

perhaps we could add something along the lines of

```
envoy --mode dump --dump components
```
potentially it could also do a `config_dump` type thing where it tells you what would be configured

just an idea, not sure if i should raise a ticket

Reply:
Yeah I agree this would be nice. Can you open a ticket to track this?